app/models/order.rb

```diff
@@ -239,9 +239,13 @@ def update_order_detail_accounts
     end
   end
 
-  def total_cost(order_detail_method)
+  def total_cost(order_detail_method, acting_user = nil)
```

Comment:
I'm not crazy about this, but the filtering does appear to need to be done (see commit c303e69509c9cff789fe542725fb080591eeb477).  And this is currently duplicating the logic in the controller action.

I'm thinking that it is probably better to just do a sum on `@order_details`'s total. Could we just have `number_to_currency(@order_details.sum { |od| od.total.to_i })` in the view (or set that total in the controller)? Having that extra wrapper class seemed a little overkill.